frontend/javascripts/admin/task/task_search_form.js

```diff
@@ -80,7 +81,11 @@ class TaskSearchForm extends React.Component<Props, State> {
         ? this.props.initialFieldValues
         : this.state.fieldValues;
     if (_.size(fieldValues) > 0) {
-      this.props.form.setFieldsValue(fieldValues);
+      const form = this.formRef.current;
+      if (!form) {
+        return;
```

Comment:
Maybe log an error here, as this behaviour is unexpected and would be silent otherwis. But I think throwing an error here or showing a toast is too much as this part only sets the initial values. It should not completely block the user.